postgres/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.1",
+  "manifest_version": "0.1.2",
```

Comment:
The field to update is actually `version`, that should be set to `2.1.1`.
Sorry for the hassle, we know this is a bit misleading and we're working to remove the version from the manifest.